Crash on rendering player heads with Fabric Language Kotlin 1.13.5 (Firmament 3.7.0+mc1.21.5)

### Firmament Version

3.7.0+ mc 1.21.5

### Bug Description

Fabric Language Kotlin 1.13.5 does not expose the kotlinx-datetime library to Firmament anymore, causing Instant to not be found at runtime. Still works with Fabric Language Kotlin 1.13.4.  Thing is, modrinth does not see it as a conflict.

### Minecraft Log

```plain text
---- Minecraft Crash Report ----
// I let you down. Sorry :(

Time: 2025-08-16 11:00:30
Description: Rendering item

java.lang.NoClassDefFoundError: kotlinx/datetime/Instant
	at knot//moe.nea.firmament.util.json.InstantAsLongSerializer.deserialize(InstantAsLongSerializer.kt:16)
	at knot//moe.nea.firmament.util.json.InstantAsLongSerializer.deserialize(InstantAsLongSerializer.kt:13)
	at knot//kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	at knot//kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
	at knot//kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
	at knot//kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
	at knot//moe.nea.firmament.util.mc.MinecraftTexturesPayloadKt$$serializer.deserialize(SkullItemData.kt:29)
	at knot//moe.nea.firmament.util.mc.MinecraftTexturesPayloadKt$$serializer.deserialize(SkullItemData.kt:29)
	at knot//kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	at knot//kotlinx.serialization.json.Json.decodeFromString(Json.kt:149)
	at knot//moe.nea.firmament.util.mc.SkullItemDataKt.decodeProfileTextureProperty(SkullItemData.kt:94)
	at knot//moe.nea.firmament.features.texturepack.CustomSkyBlockTextures.getSkullId(CustomSkyBlockTextures.kt:95)
	at knot//moe.nea.firmament.features.texturepack.CustomSkyBlockTextures.getSkullTexture(CustomSkyBlockTextures.kt:103)
	at knot//moe.nea.firmament.features.texturepack.CustomSkyBlockTextures.skullTextureCache$lambda$2(CustomSkyBlockTextures.kt:85)
	at knot//moe.nea.firmament.util.collections.WeakCache$CacheFunction$NoExtraData.invoke$lambda$0(WeakCache.kt:103)
	at knot//moe.nea.firmament.util.collections.WeakCache.getOrPut(WeakCache.kt:51)
	at knot//moe.nea.firmament.util.collections.WeakCache$CacheFunction$NoExtraData.invoke(WeakCache.kt:103)
	at knot//moe.nea.firmament.features.texturepack.CustomSkyBlockTextures.modifySkullTexture(CustomSkyBlockTextures.kt:116)
	at knot//net.minecraft.class_836.handler$bfb000$firmament$onGetRenderLayer(class_836.java:1524)
	at knot//net.minecraft.class_836.method_3578(class_836.java)
	at knot//net.minecraft.class_10513.method_65710(class_10513.java:69)
	at knot//net.minecraft.class_10513.method_65694(class_10513.java:22)
	at knot//net.minecraft.class_10444$class_10446.method_65614(class_10444.java:198)
	at knot//net.minecraft.class_10444.method_65604(class_10444.java:100)
	at knot//net.minecraft.class_332.method_51425(class_332.java:503)
	at knot//net.minecraft.class_332.method_51424(class_332.java:483)
	at knot//net.minecraft.class_332.method_51423(class_332.java:479)
	at knot//net.minecraft.class_329.method_1762(class_329.java:1310)
	at knot//net.minecraft.class_329.method_1759$mixinextras$wrapped$157(class_329.java:600)
	at knot//net.minecraft.class_329.mixinextras$bridge$method_1759$mixinextras$wrapped$157$158(class_329.java)
	at knot//net.minecraft.class_329.wrapMethod$can000$skycubed$wrapRenderItemHotbar(class_329.java:6077)
	at knot//net.minecraft.class_329.method_1759(class_329.java)
	at knot//net.minecraft.class_329.method_55805(class_329.java:545)
	at knot//net.minecraft.class_329.mddd4c73$skyblocktweaks-fabric$lambda$sbt$afterMainHud$0$2(class_329.java:5077)
	at knot//net.fabricmc.fabric.impl.client.rendering.WrappedLayer.render(WrappedLayer.java:35)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.invokeRenderInternal(class_9080.java)
	at knot//net.fabricmc.fabric.impl.client.rendering.SubLayer.render(SubLayer.java:43)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.method_55809(class_9080.java:29)
	at knot//net.minecraft.class_329.method_1753(class_329.java:235)
	at knot//net.minecraft.class_757.method_3192(class_757.java:554)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1353)
	at knot//net.minecraft.class_310.method_1514(class_310.java:936)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:506)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at com.modrinth.theseus.MinecraftLaunch.relaunch(MinecraftLaunch.java:87)
	at com.modrinth.theseus.MinecraftLaunch.main(MinecraftLaunch.java:18)
Caused by: java.lang.ClassNotFoundException: kotlinx.datetime.Instant
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:239)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	... 52 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at knot//moe.nea.firmament.util.json.InstantAsLongSerializer.deserialize(InstantAsLongSerializer.kt:16)
	at knot//moe.nea.firmament.util.json.InstantAsLongSerializer.deserialize(InstantAsLongSerializer.kt:13)
	at knot//kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	at knot//kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
	at knot//kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
	at knot//kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
	at knot//moe.nea.firmament.util.mc.MinecraftTexturesPayloadKt$$serializer.deserialize(SkullItemData.kt:29)
	at knot//moe.nea.firmament.util.mc.MinecraftTexturesPayloadKt$$serializer.deserialize(SkullItemData.kt:29)
	at knot//kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	at knot//kotlinx.serialization.json.Json.decodeFromString(Json.kt:149)
	at knot//moe.nea.firmament.util.mc.SkullItemDataKt.decodeProfileTextureProperty(SkullItemData.kt:94)
	at knot//moe.nea.firmament.features.texturepack.CustomSkyBlockTextures.getSkullId(CustomSkyBlockTextures.kt:95)
	at knot//moe.nea.firmament.features.texturepack.CustomSkyBlockTextures.getSkullTexture(CustomSkyBlockTextures.kt:103)
	at knot//moe.nea.firmament.features.texturepack.CustomSkyBlockTextures.skullTextureCache$lambda$2(CustomSkyBlockTextures.kt:85)
	at knot//moe.nea.firmament.util.collections.WeakCache$CacheFunction$NoExtraData.invoke$lambda$0(WeakCache.kt:103)
	at knot//moe.nea.firmament.util.collections.WeakCache.getOrPut(WeakCache.kt:51)
	at knot//moe.nea.firmament.util.collections.WeakCache$CacheFunction$NoExtraData.invoke(WeakCache.kt:103)
	at knot//moe.nea.firmament.features.texturepack.CustomSkyBlockTextures.modifySkullTexture(CustomSkyBlockTextures.kt:116)
	at knot//net.minecraft.class_836.handler$bfb000$firmament$onGetRenderLayer(class_836.java:1524)
	at knot//net.minecraft.class_836.method_3578(class_836.java)
	at knot//net.minecraft.class_10513.method_65710(class_10513.java:69)
	at knot//net.minecraft.class_10513.method_65694(class_10513.java:22)
	at knot//net.minecraft.class_10444$class_10446.method_65614(class_10444.java:198)
	at knot//net.minecraft.class_10444.method_65604(class_10444.java:100)
	at knot//net.minecraft.class_332.method_51425(class_332.java:503)
	at knot//net.minecraft.class_332.method_51424(class_332.java:483)
	at knot//net.minecraft.class_332.method_51423(class_332.java:479)
	at knot//net.minecraft.class_329.method_1762(class_329.java:1310)
	at knot//net.minecraft.class_329.method_1759$mixinextras$wrapped$157(class_329.java:600)
	at knot//net.minecraft.class_329.mixinextras$bridge$method_1759$mixinextras$wrapped$157$158(class_329.java)
	at knot//net.minecraft.class_329.wrapMethod$can000$skycubed$wrapRenderItemHotbar(class_329.java:6077)
	at knot//net.minecraft.class_329.method_1759(class_329.java)
	at knot//net.minecraft.class_329.method_55805(class_329.java:545)
	at knot//net.minecraft.class_329.mddd4c73$skyblocktweaks-fabric$lambda$sbt$afterMainHud$0$2(class_329.java:5077)
	at knot//net.fabricmc.fabric.impl.client.rendering.WrappedLayer.render(WrappedLayer.java:35)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.invokeRenderInternal(class_9080.java)
	at knot//net.fabricmc.fabric.impl.client.rendering.SubLayer.render(SubLayer.java:43)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.method_55809(class_9080.java:29)
	at knot//net.minecraft.class_329.method_1753(class_329.java:235)

-- Item being rendered --
Details:
	Item Type: minecraft:player_head
	Item Components: {minecraft:custom_name=>empty[style={!italic}, siblings=[literal{My Profile }[style={color=green}], literal{(Right Click)}[style={color=gray}]]], minecraft:lore=>class_9290[lines=[empty[style={!italic}, siblings=[empty[style={color=gray}], literal{Right-click to browse quests, view achievements,}[style={color=gray}]]], empty[style={!italic}, siblings=[literal{activate Network Boosters and more!}[style={color=gray}]]]], styledLines=[empty[style={color=dark_purple,!italic}, siblings=[empty[style={color=gray}], literal{Right-click to browse quests, view achievements,}[style={color=gray}]]], empty[style={color=dark_purple,!italic}, siblings=[literal{activate Network Boosters and more!}[style={color=gray}]]]]], minecraft:profile=>class_9296[name=Optional[{MINECRAFT_USERNAME}], id=Optional[{MINECRAFT_UUID}], properties={textures=[Property[name=textures, value=ewogICJ0aW1lc3RhbXAiIDogMTc1NTMzNDgyODA2NSwKICAicHJvZmlsZUlkIiA6ICI2OGMwMTI2MDhhZDk0OGQ4YWU1MzUwNjhhN2NmZGQwZCIsCiAgInByb2ZpbGVOYW1lIiA6ICJUdXJ0bGV4RDIiLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYjE4MTllYmEzZDNiYzQ5YmUxNmFkMTY1MDEwMDFmZGI1MTc4MTc0ZmI1NGQ0ZTA5M2NmZjQyOTI3ZGY1MDNmNCIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9, signature=YBURErn3WHt5mCVBzitYgn0QkahkEBeWxQcTpOhOcefy4ErW22VUWjiyaqPyi2M5IXkHJqIZkDFKiTzaUPrzoCZWIzb7t9E1tcnvaY0mFBiubkG722JRMFLwj2+LRJcp/RoGtXOzBHSwKK/WZRZhPQsMzfE7VgVo9139y7YsGsBCfIM4+lAlr5GI+ixizN81p2XCo/lO4pNuslvRoHnpGVCvZhrWePcam+iZrvhjmscnVpr232bwu+v+OolLLqJ2wwbdW6w0nszPCVw0wpxsTHjrmdIB7hm2iZ9T/CET8QfoQ48TE5KRiROO6fVB2pc5ltbIm/Vc0z5kFcxawqmQhiAfjA5+HjtRUU89lZeWTQePeoGrlyjc9/yWSXjXxYYey0UwAyOByqzcVW4fWh9DBPC4eAbds4Pv7BEUKGdAO7SGbb+fFZIYAAP2IVqJyqelJplq8tgAYcI8MuIorWcet8/sm8+VS+1rCPqTnp9GtKJM62FUlqfavQ7/Cl2Rlw5d6EFQl+S1DApnS4bTRPldIK0DYlvLea4dOjlBt8vOqgYheYO6Ce3HInzgPigLYBwrfb0BXosmHaYR1Gudj0cgKUAcz1AE+v0ts5EtxDi8EqnwATXLaAZ7lKHFUGiA6hhAB+1Hti0LW4T1ScSRNSqhB3PoIK4xE+vEtqkmYMhhobs=]]}, gameProfile=com.mojang.authlib.GameProfile@7c92e839[id={MINECRAFT_UUID},name={MINECRAFT_USERNAME},properties={textures=[Property[name=textures, value=ewogICJ0aW1lc3RhbXAiIDogMTc1NTMzNDgyODA2NSwKICAicHJvZmlsZUlkIiA6ICI2OGMwMTI2MDhhZDk0OGQ4YWU1MzUwNjhhN2NmZGQwZCIsCiAgInByb2ZpbGVOYW1lIiA6ICJUdXJ0bGV4RDIiLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYjE4MTllYmEzZDNiYzQ5YmUxNmFkMTY1MDEwMDFmZGI1MTc4MTc0ZmI1NGQ0ZTA5M2NmZjQyOTI3ZGY1MDNmNCIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9, signature=YBURErn3WHt5mCVBzitYgn0QkahkEBeWxQcTpOhOcefy4ErW22VUWjiyaqPyi2M5IXkHJqIZkDFKiTzaUPrzoCZWIzb7t9E1tcnvaY0mFBiubkG722JRMFLwj2+LRJcp/RoGtXOzBHSwKK/WZRZhPQsMzfE7VgVo9139y7YsGsBCfIM4+lAlr5GI+ixizN81p2XCo/lO4pNuslvRoHnpGVCvZhrWePcam+iZrvhjmscnVpr232bwu+v+OolLLqJ2wwbdW6w0nszPCVw0wpxsTHjrmdIB7hm2iZ9T/CET8QfoQ48TE5KRiROO6fVB2pc5ltbIm/Vc0z5kFcxawqmQhiAfjA5+HjtRUU89lZeWTQePeoGrlyjc9/yWSXjXxYYey0UwAyOByqzcVW4fWh9DBPC4eAbds4Pv7BEUKGdAO7SGbb+fFZIYAAP2IVqJyqelJplq8tgAYcI8MuIorWcet8/sm8+VS+1rCPqTnp9GtKJM62FUlqfavQ7/Cl2Rlw5d6EFQl+S1DApnS4bTRPldIK0DYlvLea4dOjlBt8vOqgYheYO6Ce3HInzgPigLYBwrfb0BXosmHaYR1Gudj0cgKUAcz1AE+v0ts5EtxDi8EqnwATXLaAZ7lKHFUGiA6hhAB+1Hti0LW4T1ScSRNSqhB3PoIK4xE+vEtqkmYMhhobs=]]}]], minecraft:custom_data=>{}, minecraft:item_model=>minecraft:player_head, minecraft:attribute_modifiers=>class_9285[modifiers=[]], minecraft:enchantments=>ItemEnchantments{enchantments={}}, minecraft:item_name=>translation{key='block.minecraft.player_head', args=[]}, minecraft:max_stack_size=>64, minecraft:break_sound=>Reference{ResourceKey[minecraft:sound_event / minecraft:entity.item.break]=class_3414[location=minecraft:entity.item.break, fixedRange=Optional.empty]}, minecraft:tooltip_display=>class_10712[hideTooltip=false, hiddenComponents={}], minecraft:repair_cost=>0, minecraft:equippable=>class_10192[slot=HEAD, equipSound=Reference{ResourceKey[minecraft:sound_event / minecraft:item.armor.equip_generic]=class_3414[location=minecraft:item.armor.equip_generic, fixedRange=Optional.empty]}, assetId=Optional.empty, cameraOverlay=Optional.empty, allowedEntities=Optional.empty, dispensable=true, swappable=false, damageOnHurt=true, equipOnInteract=false], minecraft:rarity=>UNCOMMON}
	Item Foil: false
Stacktrace:
	at knot//net.minecraft.class_332.method_51425(class_332.java:503)
	at knot//net.minecraft.class_332.method_51424(class_332.java:483)
	at knot//net.minecraft.class_332.method_51423(class_332.java:479)
	at knot//net.minecraft.class_329.method_1762(class_329.java:1310)
	at knot//net.minecraft.class_329.method_1759$mixinextras$wrapped$157(class_329.java:600)
	at knot//net.minecraft.class_329.mixinextras$bridge$method_1759$mixinextras$wrapped$157$158(class_329.java)
	at knot//net.minecraft.class_329.wrapMethod$can000$skycubed$wrapRenderItemHotbar(class_329.java:6077)
	at knot//net.minecraft.class_329.method_1759(class_329.java)
	at knot//net.minecraft.class_329.method_55805(class_329.java:545)
	at knot//net.minecraft.class_329.mddd4c73$skyblocktweaks-fabric$lambda$sbt$afterMainHud$0$2(class_329.java:5077)
	at knot//net.fabricmc.fabric.impl.client.rendering.WrappedLayer.render(WrappedLayer.java:35)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.invokeRenderInternal(class_9080.java)
	at knot//net.fabricmc.fabric.impl.client.rendering.SubLayer.render(SubLayer.java:43)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.method_55809(class_9080.java:29)
	at knot//net.minecraft.class_329.method_1753(class_329.java:235)
	at knot//net.minecraft.class_757.method_3192(class_757.java:554)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1353)
	at knot//net.minecraft.class_310.method_1514(class_310.java:936)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:506)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at com.modrinth.theseus.MinecraftLaunch.relaunch(MinecraftLaunch.java:87)
	at com.modrinth.theseus.MinecraftLaunch.main(MinecraftLaunch.java:18)

-- Uptime --
Details:
	JVM uptime: 43.227s
	Wall uptime: 33.491s
	High-res time: 21.386s
	Client ticks: 247 ticks / 12.350s
Stacktrace:
	at knot//net.minecraft.class_310.method_1587(class_310.java:2382)
	at knot//net.minecraft.class_310.method_54580(class_310.java:1005)
	at knot//net.minecraft.class_310.method_1514(class_310.java:957)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:506)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at com.modrinth.theseus.MinecraftLaunch.relaunch(MinecraftLaunch.java:87)
	at com.modrinth.theseus.MinecraftLaunch.main(MinecraftLaunch.java:18)
```

--- a/src/main/kotlin/apis/Profiles.kt
+++ b/src/main/kotlin/apis/Profiles.kt
@@ -6,7 +6,7 @@ package moe.nea.firmament.apis
 
 import io.github.moulberry.repo.constants.Leveling
 import io.github.moulberry.repo.data.Rarity
-import kotlinx.datetime.Instant
+import java.time.Instant
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.UseSerializers

--- a/src/main/kotlin/util/json/InstantAsLongSerializer.kt
+++ b/src/main/kotlin/util/json/InstantAsLongSerializer.kt
@@ -2,7 +2,7 @@
 
 package moe.nea.firmament.util.json
 
-import kotlinx.datetime.Instant
+import java.time.Instant
 import kotlinx.serialization.KSerializer
 import kotlinx.serialization.descriptors.PrimitiveKind
 import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
@@ -13,10 +13,10 @@ import kotlinx.serialization.encoding.Encoder
 object InstantAsLongSerializer : KSerializer<Instant> {
     override val descriptor: SerialDescriptor = PrimitiveSerialDescriptor("InstantAsLongSerializer", PrimitiveKind.LONG)
     override fun deserialize(decoder: Decoder): Instant {
-        return Instant.fromEpochMilliseconds(decoder.decodeLong())
+        return Instant.ofEpochMilli(decoder.decodeLong())
     }
 
     override fun serialize(encoder: Encoder, value: Instant) {
-        encoder.encodeLong(value.toEpochMilliseconds())
+        encoder.encodeLong(value.toEpochMilli())
     }
 }

--- a/src/main/kotlin/util/mc/SkullItemData.kt
+++ b/src/main/kotlin/util/mc/SkullItemData.kt
@@ -5,9 +5,8 @@ package moe.nea.firmament.util.mc
 import com.mojang.authlib.GameProfile
 import com.mojang.authlib.minecraft.MinecraftProfileTexture
 import com.mojang.authlib.properties.Property
+import java.time.Instant
 import java.util.UUID
-import kotlinx.datetime.Clock
-import kotlinx.datetime.Instant
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.UseSerializers
 import net.minecraft.component.DataComponentTypes
@@ -32,7 +31,7 @@ data class MinecraftTexturesPayloadKt(
     val profileId: UUID? = null,
     val profileName: String? = null,
     val isPublic: Boolean = true,
-    val timestamp: Instant = Clock.System.now(),
+    val timestamp: Instant = Instant.now(),
 )
 
 fun GameProfile.setTextures(textures: MinecraftTexturesPayloadKt) {

